api/apps/api/src/modules/scenarios/scenarios-crud.service.ts

```diff
@@ -276,4 +283,21 @@ export class ScenariosCrudService extends AppBaseService<
       : undefined;
     return wdpaAreaIdsWithinPlanningArea;
   }
+
+  async extendFindAllQuery(
+    query: SelectQueryBuilder<Scenario>,
+    fetchSpecification: FetchSpecification,
+    info?: ScenarioInfoDTO,
+  ): Promise<SelectQueryBuilder<Scenario>> {
+    const nameAndDescriptionFilter = info?.params?.nameAndDescriptionFilter;
+    if (nameAndDescriptionFilter) {
+      const nameAndDescriptionFilterField = 'nameAndDescriptionFilter' as const;
+      query.andWhere(
+        `(${this.alias}.name ||' '|| COALESCE(${this.alias}.description, '')) ILIKE :${nameAndDescriptionFilterField}`,
+        { [nameAndDescriptionFilterField]: `%${nameAndDescriptionFilter}%` },
+      );
```

Comment:
instead of `ILIKE` a text search operator could be used `@@`, but I wasn't sure about the scope of the PR.

Reply:
@Dyostiq `ILIKE` should be ok here